### PR TITLE
"xhr" -> "XMLHttpRequest" where appropriate

### PR DESCRIPTION
**Pull request by TehShrike**
Changed references to `xhr` to `XMLHttpRequest` where they were referencing the web API instead of this module.

### CODE REVIEW

**Comment by TehShrike** [comment]
Even though #116 was a different issue, I think this is still a good change.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # xhr
 
-A small xhr wrapper. Designed for use with [browserify](http://browserify.org/).
+A small XMLHttpRequest wrapper. Designed for use with [browserify](http://browserify.org/).
```
Comment (yoshuawuyts) [inline README.md]: Go all out and also link to MDN? - https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest

Comment (TehShrike) [inline README.md]: I wouldn't be against it.  I was mostly just aiming to reduce the name confusion with this pull request (there's already a ton of links to XMLHttpRequest docs/specs throughout the readme).

Comment (yoshuawuyts) [inline README.md]: cool yeah, no pressure - patch LGTM regardless

**Pull request merged**